src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -175,27 +287,97 @@ public boolean isRecognizedFormat(BufferedReader reader) throws IOException {
 
     private HashMap<String, Field> getFieldMappings() {
         HashMap<String, Field> fieldMappings = new HashMap<>();
```

Comment:
Make map `public static final` to avoid initialization at each import.

(This also removes the method)